### DISCUSSION TITLE
Auto sliding of image or text like marquee functionality

### DISCUSSION THREAD

**Discussion by RyanAlgo** [elementor/elementor#22193] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Some design uses that function and I've always looked at and done it on CSS/JS. But it might be a great help if elementor pro have that.

### Describe the solution you'd like

on the slider widget, if there is an option to make it like that instead of sliding 1 by 1, there's an option to continuously slide.

**Comment by SolunaCG** [comment]
+1

**Comment by datyorubadev** [comment]
We definitely needed marquee to be added as a widget in Elementor, which will support both text, image and text-image functionality. We will be looking forward to it in the next update.
